SharpCompress.sln

```diff
@@ -1,44 +1,44 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
The solution file was regenerated by Visual Studio. I don't see any real changes.